src/bindings/capi/tvgCapi.cpp

```diff
@@ -162,6 +162,17 @@ TVG_EXPORT Tvg_Result tvg_paint_transform(Tvg_Paint* paint, const Tvg_Matrix* m)
 }
 
 
+TVG_EXPORT Tvg_Result tvg_paint_get_transform(Tvg_Paint* paint, Tvg_Matrix* m)
+{
+    if (!paint || !m) return TVG_RESULT_INVALID_ARGUMENT;
+    tvg::Matrix mTranfs = reinterpret_cast<Paint*>(paint)->transform();
+    m->e11 = mTranfs.e11; m->e12 = mTranfs.e12; m->e13 = mTranfs.e13;
```

Comment:
I think you can cast the matrix simply like this.
*m = (Tvg_Matrix) reinterpret_cast<Paint*>(paint)->transform();